dietstory-routing/dietstory-routing/settings.py

```diff
@@ -161,6 +161,8 @@
 EMAIL_HOST_PASSWORD = secret_config['EMAIL_HOST_PASSWORD']
 EMAIL_USE_TLS = bool(secret_config['EMAIL_USE_TLS'])
 
+DIETSTORY_API_HOST = 'http://' + secret_config['DIETSTORY_API_HOST']
```

Comment:
is `http://` necessary?

Reply:
Yea, otherwise the requests library doesn't know what type of protocol to use. I can move it to the config level if you prefer.

Reply:
good to know - putting it here is fine.